cg/meta/workflow/balsamic.py

```diff
@@ -37,16 +41,19 @@ def __init__(
         fastq_handler: FastqHandler,
         lims_api: LimsAPI,
         trailblazer_api: TrailblazerAPI,
+        hermes_api: HermesApi,
     ):
         self.balsamic_api = balsamic_api
         self.store = store
         self.housekeeper_api = housekeeper_api
         self.fastq_handler = fastq_handler
         self.lims_api = lims_api
         self.trailblazer_api = trailblazer_api
+        self.hermes_api = hermes_api
 
-    def get_case_object(self, case_id: str):
+    def get_case_object(self, case_id: str) -> models.Family:
         """Look up case ID in StoreDB and return result"""
+        LOG.info("Fetching case %s", case_id)
```

Comment:
This might get annoying as its being done all the time. Maybe put in log.debug?

Reply:
Anoying for who?

Reply:
But sure I can use debug

Reply:
Changed to debug